host_core/lib/host_core/wasmcloud/runtime.ex

```diff
@@ -40,6 +40,15 @@ defmodule HostCore.WasmCloud.Runtime do
     end
   end
 
+  @doc ~S"""
+  Initializes the `HostCore.WasmCloud.Runtime` by e.g. setting NIF-wide logger
```

Comment:
see previous comment. Nothing should be NIF-wide, we should rather have a per-runtime-instance initialization. It may not seem like it at first glance, but the boundary on the `Vhost.VirtualHost` GenServer process needs to be a hard boundary.